tests/run_examples.py

```diff
@@ -149,6 +149,8 @@ def check_library(rule):
 req_version["knn_bf_regression_spmd.py"] = (2023, "P", 100)
 req_version["linear_regression_spmd.py"] = (2023, "P", 100)
 req_version["logistic_regression_spmd.py"] = (2024, "P", 400)
+# Timeout on PVC, bumped the req version to deselect
+req_version["sycl/gradient_boosted_regression.py"] = (2024, "P", 500)
```

Comment:
Could bump this to 600 since we will be at 2024.5 once https://github.com/oneapi-src/oneDAL/pull/2730 is merged